functionsForFunSwift.playground/Contents.swift

```diff
@@ -17,8 +17,35 @@ enum DriveTrain {
 struct Vehicle {
     let information: VehicleInformation
     let driveTrain: DriveTrain
+    
+    func vehicleOpinion() {
+        switch driveTrain {
+        case .awd: print("Ohhh, you have a \(information.Model). That's an \(DriveTrain.awd.rawValue), so you can go off-roading with it!")
+        case .rwd: print("If you want fast car, a \(DriveTrain.rwd.rawValue) is what you want. So you'll have fun with your \(information.Model) on the road.")
```

Comment:
I just realized that the properties in VehicleInformation should begin with a lower case!